### PR TITLE
WIP:[PIE-6] Added initial infrastructure for front-end

### PR DESCRIPTION
**Pull request by capmorganbih**
"**front-end**" directory was added. The directory contains "**static**" folder with static assets and "**infrastructure**" folder with aws-cdk code written in typescript. The infrastructure contains only S3 bucket with public access for "**getObject**" action.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,13 @@
 # FoodBridge
+FoodBridge is a next generation food delivery application.
+
+### How to deploy
+1. Set up environment variables.:
```
Comment (NatalyBV) [inline README.md:5]: Remove . dot before colon.

File: front-end/infrastructure/package.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "infrastructure",
+  "version": "0.1.0",
```
Comment (NatalyBV) [inline front-end/infrastructure/package.json:3]: Why isn't it SNAPSHOT?

File: front-end/infrastructure/lib/front-end-stack.ts [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+    this.staticAssetsBucket = new Bucket(this, 'static-assets-bucket', {
+      bucketName: `food-bridge-statics-${Properties.ENVIRONMENT}`,
+      removalPolicy: RemovalPolicy.RETAIN,
+      publicReadAccess: true,
+      websiteIndexDocument: 'index.html',
+      websiteErrorDocument: '404error.html',
+    });
+
+    new Asset(this, 'site-statics', {
```
Comment (NatalyBV) [inline front-end/infrastructure/lib/front-end-stack.ts:23]: The destination bucket isn't specified

**Review by NatalyBV — CHANGES_REQUESTED** [review]
Change these )

File: README.md (hunk 1 above)
Comment (capmorganbih) [reply README.md:5]: Fixed.

File: front-end/infrastructure/package.json (hunk 2 above)
Comment (capmorganbih) [reply front-end/infrastructure/package.json:3]: It should not be SNAPSHOT

File: front-end/infrastructure/lib/front-end-stack.ts (hunk 3 above)
Comment (capmorganbih) [reply front-end/infrastructure/lib/front-end-stack.ts]: Fixed

**Review by NatalyBV — APPROVED** [review]
Amazing!

**Pull request merged**